comment/managers/comments.py

```diff
@@ -53,3 +55,25 @@ def generate_urlhash(self):
             len_id=settings.COMMENT_URL_ID_LENGTH,
             suffix=settings.COMMENT_URL_SUFFIX
             )
+
+    def get_parent_comment(self, parent_id):
+        parent_comment = None
+        if parent_id:
+            parent_qs = self.filter(id=parent_id)
+            if parent_qs.exists():
+                parent_comment = parent_qs.first()
+        return parent_comment
+
+    def get_dict(self, instance):
+        opts = instance._meta
+        data = {}
+        for f in chain(opts.concrete_fields, opts.private_fields):
+            if getattr(f, 'ct_field', None):
+                continue
+            data[f.name] = f.value_from_object(instance)
+        for f in opts.many_to_many:
+            data[f.name] = [i.id for i in f.value_from_object(instance)]
+        return data
```

Comment:
I'll remove this method, this isn't used now.

`content_type` and `object_id` can't be injected directly to the `Comment` model. We also need `app_name`, `model_name`, `model_id` before obtaining the `content_object`, I thought it would be better to personally form the dictionary in the view method.

This will be removed in the final commit